### PR TITLE
Release 2.2.0 and 2.2.1

### PR DESCRIPTION
**Pull request by rb3ckers**

### CODE REVIEW

File: setup/otel/collector.md [hunk 1]
```diff
@@ -161,7 +165,7 @@ The `service` section determines what components of the collector are enabled. T
...
 * `processors`: The `memory_limiter` helps to prevent out-of-memory errors. The `batch` processor helps better compress the data and reduce the number of outgoing connections required to transmit the data. The `resource` processor adds additional resource attributes (discussed separately)
-* `exporters`: The `debug` exporter simply logs to stdout which helps when troubleshooting. The `otlp/stackstate` exporter sends telemetry data to SUSE Observability using the OTLP protocol. It is configured to use the bearertokenauth extension for authentication to send data to the SUSE Observability OTLP endpoint.
+* `exporters`: The `debug` exporter simply logs to stdout which helps when troubleshooting. The `otlp/stackstate` exporter sends telemetry data to SUSE Observability using the OTLP protocol via GRPC (Default). The `otlphttp/stackstate` exporter sends telemetry data to SUSE Observability using the OTLP protocol via HTTP and is meant to be used where there area some impediments to use the GRPC one (Needs to be activated in the pipelineS). Both OTLP exporters are configured to use the bearertokenauth extension for authentication to send data to the SUSE Observability OTLP endpoint.
```
Comment (fvlankvelt) [inline setup/otel/collector.md]: some funny CaPitaliZatIon going on

File: setup/security/authentication/oidc.md [hunk 2]
```diff
@@ -41,12 +41,11 @@ stackstate:
         access_type: offline
 
     # map the groups from OIDC provider
-    # to the 4 standard roles in SUSE Observability (guest, powerUser, admin and platformAdmin)
+    # to the 4 standard roles in SUSE Observability (guest, powerUser and admin)
```
Comment (fvlankvelt) [inline setup/security/authentication/oidc.md]: there's only 3

**Review by aacevedoosorio — DISMISSED** [review]

File: setup/security/authentication/single_password.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+description: SUSE Observability Self-hosted
+---
+
+# Single password
+
+Out of the box, SUSE Observability is configured with a single password configuration, [configured during installation](/k8s-suse-rancher-prime.md#installation). This authenticates users with a single, randomly generated password created by the `sus-observability-values` chart.
```
Comment (fvlankvelt) [inline setup/security/authentication/single_password.md]: typo in `suse-observability-values`

File: setup/security/authentication/oidc.md (hunk 2 above)
Comment (rb3ckers) [reply setup/security/authentication/oidc.md]: k8s-troubleshooter is also there, but not in this list for some reason :(. Great.

**Review by fvlankvelt — DISMISSED** [review]

**Review by fvlankvelt — APPROVED** [review]

**Pull request merged**